cron/readme.md

```diff
@@ -0,0 +1,20 @@
+# Run things periodically, a la cron
+
+This image runs user-supplied commands at regular intervals, similar
+to the [`cron(8)`][cron-8] utility.
+
+Drop your executable scripts in any of the usual `/etc/cron.*/*`
+directories (which are wiped off all the standard Debian stuff on
+build). You should probably do so by extending this image with your
+custom code / services.
+
+## Running your code
+
+Supported directories / schedules are:
+
+- `/etc/cron.hourly` - Every hour at 17th minute
+- `/etc/cron.daily` - Every 24 hours at 6:25 AM
+- `/etc/cron.weekly` - Every Saturday at 6:47 AM
```

Comment:
See note on https://github.com/unit9/docklabs/pull/25/commits/3b1e442a3c0f4c44d4dac2d5b3149846e412e381

Perhaps better not to implement these at all